liquigraph-core/src/main/java/org/liquigraph/core/configuration/ConfigurationBuilder.java

```diff
@@ -98,6 +102,15 @@ public ConfigurationBuilder withUsername(String username) {
     }
 
     /**
+     * Specifies the default username as the username allowed to connect to the remote
+     * graph database instance.
+     * @return itself for chaining purposes
+     */
+    public ConfigurationBuilder withDefaultUsername() {
```

Comment:
After thinking about it again and discussing with Frank, I think we can somehow alleviate the problem by validating that config with (user XOR password) is a faulty case. 

Either both are specified or none of them.

Failing to comply with this would then result in a simple validation message, without resorting to tweaking exception handling etc. 

In the end, I think that would cover the initial scenario: if a user forgets to specify a username (because he/she thinks a default one is provided, an explicit error msg will be displayed).

Does it make sense?
Sorry for the back and forth on this, thanks a lot for your patience!